src/Sarif/Autogenerated/LocationEqualityComparer.cs

```diff
@@ -38,6 +38,11 @@ public bool Equals(Location left, Location right)
                 return false;
             }
 
+            if (left.LogicalLocationIndex != right.LogicalLocationIndex)
+            {
+                return false;
+            }
```

Comment:
>} [](start = 12, length = 1)

This is relevant to your comment in the PR description: that two locations with different indices in different log files might actually turn out to be the same. As I continue the review I expect to see how we avoid this potentially incorrect generated behavior.

Reply:
Someone should explicitly review and confirm this in todo: [https://github.com/Microsoft/sarif-sdk/issues/1174](https://github.com/Microsoft/sarif-sdk/issues/1174)

---
In reply to: [243719244](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243719244) [](ancestors = 243719244)